Cargo.toml

```
@@ -6,17 +6,19 @@ license = "MIT OR Apache-2.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+default = ["clap"]
```

nitpick but imo it is nicer if the feature a user is selecting is "cli" or something a bit more specific than just "clap"

```suggestion
default = ["cli"]
cli = ["clap"]
```